r/tests/testthat/test-dplyr-across.R

```diff
@@ -279,7 +277,17 @@ test_that("purrr-style lambda functions are supported", {
   )
 })
 
-test_that("ARROW-14071 - function(x)-style lambda functions are not supported", {
+test_that("ARROW-14071 - user-defined R functions", {
```

Comment:
```suggestion
test_that("ARROW-14071 - R functions from a user's environment", {
```

Just to be super clear this _isn't_ about UDFs